lib/gui/pages/main/controllers/image-selection.js

```diff
@@ -65,6 +73,37 @@ module.exports = function(SupportedFormatsModel, SelectionStateModel, AnalyticsS
       return;
     }
 
+    if (SupportedFormatsModel.seemsWindowsImage(image.path)) {
+
+      AnalyticsService.logEvent('Possibly Windows image', image);
+
+      WarningModalService.display({
+        confirmationLabel: 'Continue',
+        rejectionLabel: 'See documentation',
+        description: messages.warning.possiblyWindowsImage()
+      }).then((userAccepted) => {
+        if (userAccepted) {
+          AnalyticsService.logEvent('User accepted possibly Windows image', image);
+          this.finalizeImageSelection(image);
+        } else {
+          AnalyticsService.logEvent('User rejected possibly Windows image', image);
+          OSOpenExternalService.open('https://github.com/resin-io/etcher/blob/master/docs/USER-DOCUMENTATION.md#why-is-my-drive-not-bootable');
+        }
+      });
+
+    } else {
+      this.finalizeImageSelection(image);
```

Comment:
After the following PR https://github.com/resin-io/etcher/pull/853 the contents of this function will be reduced to just two lines, so I guess let's just inline it here.